### ISSUE TITLE
Add "connect with us" section to devhub landing page

### ISSUE THREAD

**Issue by eviljeff** [mozilla/addons-server#12613]
Extension workshop (and the invision mocks) for the new devhub landing page shown to unauthenticated users has a section promoting our developer facing social media and a newletter signup box.  We need to add that section to the devhub landing page also.

![Screenshot 2019-10-15 11 01 19](https://user-images.githubusercontent.com/768592/66822104-7c5ba300-ef3b-11e9-9400-e6d7c7644df3.png)

**Comment by muffinresearch** [comment]
See https://github.com/mozilla/basket-example for a client example.

**Comment by AlexandraMoga** [comment]
I've verified the Connect with us section on -dev and stage in terms of content, styling, layout, newsletter subscription.
I found some issues for which I filed follow-ups. 

@eviljeff 
I have an observation:

the section has been implemented only for the logged out view, which is also what was proposed in the mocks. However, there is a comment from @jvillalobos in [this doc](https://docs.google.com/document/d/1CPZUrNxhdGIXyF6PTI5L9BMBp8jgrxuKHTSJoUvZgxk/edit) which says:
>The logged in view should have the edited grey footer, blue footer, and black footer.

By 'blue footer' I understand that the Connect with us section should be present in the logged in view as well, since it offers access to resources present nowhere else on the page. @jvillalobos to confirm if this is indeed expected.
